2020.11 docs (#3053)
biocore/qiita: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst

#. Metatranscriptome sequencing

> Note that the selected processing are mainly guided so we can perform meta-analyses,

Reviewer: ```suggestion
Note that the selected processing recommendations are mainly guided towards performing meta-analyses,
```

Shotgun sequencing
------------------

Qiita currently has one active shotgun metagenomics data analysis pipeline: a per sample

> bowtie2 alignment step with Woltka classification using either the WoLr1 or Rep200 databases.

Reviewer: What's `WoLr1`?

Author: WoLr1 ("Web of Life" release 1)
